### ISSUE TITLE
Support TS for indexTransform

### ISSUE THREAD

**Issue by at-vinguyen** [just-jeb/angular-builders#687]
Now, I can't import **_ts file_** in `index-transform.js`, see the code snippet below (just in case):
```js
import { environment } form 'environment.ts'
module.exports = (targetOptions, indexHtml) => {
  // use environment info to do some things
}
```
So I think it's better if TS is supported for indexTransform then we can use TypeScipt great features if needed.

**Comment by just-jeb** [comment]
[Related](https://github.com/just-jeb/angular-builders/issues/681#issuecomment-588302447)

**Comment by vixnguyen** [comment]
@just-jeb 
Can you allow me to create a new PR to solve this issue?

**Comment by just-jeb** [comment]
Sure, go for it!

**Comment by vixnguyen** [comment]
It seems I don't have permission to push my branch 🤔

**Comment by just-jeb** [comment]
You should create a fork from this repo, implement the changes there and create a PR from the fork.

**Comment by vixnguyen** [comment]
Already finished. Please help me to check 🎉

**Comment by just-jeb** [comment]
Closed by #689
